docs/VLCB4in4out_README.md

```diff
@@ -173,40 +191,7 @@ The FCU will show a new event produced by the Software Node in the VLCB4in4out e
 table.  It will not, for some reason, remove the now gone original event automatically.
 It is necessary to highlight the redundant event and use alt-D to remove it.
 
-## Serial Monitor
-
-By sending a single character to the Arduino using the serial send facility in the
-serial monitor of the Arduino IDE (or similar),it is possible to initiate certain operations
-or get information from the Arduino sketch.
-
-#### 'r' (implementation pending)
-This character will cause the module to renegotiate its VLCB status by requesting a node number.
-The FCU will respond as it would for any other unrecognised module.
-
-#### 'z'
-This character needs to be sent twice within 2 seconds so that its action is confirmed.
-This will reset the module and clear the EEPROM.  It should thus be used with care.
-
-Other information is available using the serial monitor using other commands:
 
-#### 'n'
-This character will return the node configuration.
-
-#### 'e'
-This character will return the learned event table in the EEPROM.
-
-#### 'v'
-This character will return the node variables.
-
-#### 'c'
-This character will return the CAN bus status.
-
-#### 'h'
-This character will return the event hash table.
-
-#### 'm'
-This character will return the amount of free memory. 
- 
```

Comment:
Oh, need to capture these in some other documentation file before they disappear from this file.

Reply:
Serial User Interface document now added with this information.